apps/coscrad-frontend/src/components/higher-order-components/selected-categorizables-of-multiple-types.presenter.tsx

```diff
@@ -33,9 +32,9 @@ export const SelectedCategorizablesOfMultipleTypesPresenter = ({
 }: SelectedCategorizablesOfMultipleTypesPresenterProps): JSX.Element => {
     return (
         <div data-testid="multiple-categorizables-view">
-            <Typography variant="h3">
+            {/* <Typography variant="h4">
```

Comment:
heading property - ideally would be in the Layout file.  Are there going to be other connected categorizables of multiple types to display in this panel?

Reply:
This is used for displaying
- all self-notes for a given resource
- all connections for a given resource
- all categorizables with a given tag
- eventually all resources belonging to a given category node